tests/WebAppCallsMicrosoftGraph/Startup.cs

```diff
@@ -74,6 +91,21 @@ public void ConfigureServices(IServiceCollection services)
             }).AddMicrosoftIdentityUI();
         }
 
+        /// <summary>
+        /// Prototype of certificate-less authentication using a signed assertion
+        /// acquired with MSI (federated identity)
+        /// </summary>
+        /// <returns></returns>
+        private async Task<ClientAssertion> GetSignedAssertionFromMsi(CancellationToken cancellationToken)
+        {
```

Comment:
UserAssignedClientId is available in MsIdentityOptions.

Reply:
Thanks for the reminder. Fixed this